image/templates/helm/shared/internal/scanner-v4-config-shape.yaml

```diff
@@ -0,0 +1,81 @@
+scannerV4:
+  disable: null # bool
+  indexer:
+    metricsPort: null # int
+    disable: null # bool
+    logLevel: null # string
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    replicas: null # int
+    autoscaling:
+      disable: null # bool
+      minReplicas: null # int
+      maxReplicas: null # int
+    nodeSelector: null # string | dict
+    tolerations: null # [dict]
+    affinity: null # dict
+    resources: null # string | dict
+    serviceTLS:
+      cert: null # string
+      key: null # string
+      generate: null # bool
+  matcher:
+    metricsPort: null # int
+    disable: null # bool
+    logLevel: null # string
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    replicas: null # int
+    autoscaling:
+      disable: null # bool
+      minReplicas: null # int
+      maxReplicas: null # int
+    nodeSelector: null # string | dict
+    tolerations: null # [dict]
+    affinity: null # dict
+    resources: null # string | dict
+    serviceTLS:
+      cert: null # string
+      key: null # string
+      generate: null # bool
+  db:
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    source:
+      minConns: null # int
+      maxConns: null # int
+      statementTimeoutMs: null # int
+    configOverride: null # string
+    password:
```

Comment:
I would rename this to `initialPassword` since it would only be used at creation time

Reply:
I think Central DB and the original Scanner DB use `password`, so let's keep it as-is for consistency